### ISSUE TITLE
How to replace JHU data with country-wise data (India) and error in cleaning country level datasets

### ISSUE THREAD

**Issue by SM-ins** [lisphilar/covid19-sir#120]
Dear @lisphilar ,

while running the following code,i am getting mentioned error:-

ind_data = cs.CountryData("/kaggle/input/covid19-in-india/covid_19_india.csv",Country="India")
ind_data.set_variables(
    date="Date", confirmed="Positive", fatal="Fatal", recovered="Discharged", province=None
)
ind_data.cleaned().tail()

TypeError: __init__() got an unexpected keyword argument 'Country'

I just replace the country by India in place of japan and change the file path too...,but in spite of that I am getting above error..,will you please help..

With regards,
Rakesh

**Comment by lisphilar** [comment]
Dear Rakesh,
Please use small characters as the arguments of methods like this.
```Python
ind_data = cs.CountryData("/kaggle/input/covid19-in-india/covid_19_india.csv",country="India")
```

Before: Country="India"
After: country="India"

Thank you.

**Comment by lisphilar** [comment]
Dear Rakesh,
Please read documentation of CountryData.set_variables()
https://lisphilar.github.io/covid19-sir/covsirphy.cleaning.html#covsirphy.cleaning.country_data.CountryData.set_variables

(If you find errors or something to know, please let me know as always.)

Did you used the following dataset?
https://www.kaggle.com/sudalairajkumar/covid19-in-india?select=covid_19_india.csv

I think the following codes will be necessary.
```Python
ind_data.set_variables(
date="Date", confirmed="Confirmed", fatal="Deaths", recovered="Cured", province="State/UnionTerritory"
)
```

**Comment by SM-ins** [comment]
Dear @lisphilar/covid19-sir
<reply@reply.github.com> ,

Yes..,you are right..,the given code is necessary as column name were not
matching..,thanks for the help..

With regards,
Rakesh

**Comment by lisphilar** [comment]
Thank you!
I will close this issue.

**Comment by SM-ins** [comment]
Dear Lisphilar,

i am agin encountering error in in running the above code:-
ind_data = cs.CountryData("/kaggle/input/covid19-in-india/covid_19_india.csv",country="India")
ind_data.set_variables(
date="Date", confirmed="Confirmed", fatal="Deaths", recovered="Cured", province="State/UnionTerritory"
)
ind_data.cleaned().tail()

ValueError: Cannot convert non-finite values (NA or inf) to integer
Please look into it....

With regards,
Rakesh

**Comment by lisphilar** [comment]
Dear Rakesh,
It seems that NA values were added to the raw dataset.
I will add `.fillna(0)` in cleaning method of`cs.CountryData` class.

**Comment by SM-ins** [comment]
Dear Lisphilar,
Thanks for the response...,okey..kindly let me know after adding it..

**Comment by lisphilar** [comment]
Dear Rakesh,
I fixed this in version 2.6.0-alpha.fix.120
Please confirm it.
